src/Stache/Query/TermQueryBuilder.php

```diff
@@ -176,4 +176,21 @@ protected function getWhereColumnKeyValuesByIndex($column)
 
         return $items;
     }
+
+    protected function getBlueprintsForRelations()
+    {
+        $taxonomies = empty($this->taxonomies)
+            ? Facades\Taxonomy::handles()
+            : $this->taxonomies;
```

Comment:
I believe this needs to be `: collect($this->taxonomies)` as flatMap is being called below and taxonomies is an array.